doit/cmdparse.py

```diff
@@ -4,7 +4,7 @@
 """
 import getopt
 import six
-
```

Comment:
I tend to use more blank lines than usual. IMO this increases readability... please dont remove my blank lines :D